pages/Embed Media - Audio, Photos, Videos.md

```diff
@@ -60,10 +66,13 @@
 				  https://logseq.github.io/screenshots/1.png
 				  ```
 				  #+BEGIN_TIP
-				  	  You can specify the image size by formatting the image link as such: `![](https://logseq.github.io/screenshots/1.png){:height 469, :width 819}`
+				  You can specify the image size by formatting the image link as such: `![](https://logseq.github.io/screenshots/1.png){:height 469, :width 819}`
 				   #+END_TIP
 - ## Videos
-	- Using [[Hiccup]] and certain slash commands, it is possible to embed video to your page and play it back.
+	- You can also easily embed and play back video files by using the `{{video }}` command, and pasting the link to the file inside the brackets after `video`:
```

Comment:
`{{ video}}` is a macro but I don't mind fixing wording

Reply:
Yes, feel free to correct the wording! Thanks for flagging this :)